src/Features/ERDDAP/Map/index.tsx

```diff
@@ -6,42 +6,30 @@ import VectorLayer from "ol/layer/Vector"
 import { AttributionLike } from "ol/source/Source"
 import VectorSource from "ol/source/Vector"
 import { Circle, Fill, Stroke, Style } from "ol/style"
-import * as React from "react"
-import { connect } from "react-redux"
-import { bindActionCreators, Dispatch } from "redux"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)